src/components/common/ChattingInput/ChattingInput.tsx

```diff
@@ -0,0 +1,80 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import type { InputHTMLAttributes, MouseEventHandler } from 'react';
+
+import { IconCamera } from '../Icons/IconCamera';
+import { Text } from '../Text';
+
+type ChattingInputProps = {
+  isChatAlive: boolean;
+  onClickSendButton: MouseEventHandler<HTMLButtonElement>;
+} & InputHTMLAttributes<HTMLInputElement>;
+
+export const ChattingInput = ({
+  isChatAlive,
+  value,
+  onClickSendButton,
+  ...props
+}: ChattingInputProps) => {
+  const sendButtonColor = isChatAlive ? (!!value ? 'secondary' : 'gray400') : 'gray500';
+
+  return (
+    <StyledChattingInputWrapper>
+      <StyledCameraButton isChatAlive={isChatAlive}>
+        <IconCamera />
+      </StyledCameraButton>
+      <StyledChattingInput
+        value={isChatAlive ? value : ''}
+        placeholder={isChatAlive ? '내용을 입력해주세요.' : '채팅방이 폭파되었습니다.'}
+        disabled={!isChatAlive}
```

Comment:
```suggestion
        disabled={isChatAlive === false}
```
이런 식으로 하면 DX가 향상될 수 있을 것 같아요!

혹은 isChatAlive === false 자체를 추상화 시켜도 좋을 것 같아요

제안입니다!

Reply:
ㅇㅎ 더 명확하네요. 알겠습니다